third_party/amd/lib/TritonAMDGPUToLLVM/BuiltinFuncToLLVM.cpp

```diff
@@ -25,8 +25,10 @@ class CallOpConversion : public mlir::RewritePattern {
   matchAndRewrite(mlir::Operation *op,
                   mlir::PatternRewriter &rewriter) const override {
     auto callOp = cast<LLVM::CallOp>(op);
-    if (isPredicatedLoad(callOp)) {
-      return convertPredicatedLoad(callOp, rewriter);
+    if (isPredicatedLoadNT(callOp)) {
```

Comment:
Super nit: leave the normal load as the first case?

Reply:
We can't. If we put normal load first, the NT load will also be detected as normal load, since it's doing substring match.

Reply:
Okay that makes sense.